### ISSUE TITLE
Can Audio/Video Profiles be transferred

### ISSUE THREAD

**Issue by Theli93** [staxrip/staxrip#1475]
This is a combination question and/or request.

Relating back to my earlier errant issue in [#1474](https://github.com/staxrip/staxrip/issues/1474#issue-2511928889) [Thanks BTW]

I've installed v2.42.0 with fresh settings as directed.
I've copied over my project templates with no apparent issues. However, many of my templates use customized Audio and/or Video Profiles,, which are now absent. I tend to retain the default ones unmodified (for basline/reference) and save my modified ones with new descriptive names for future use.

1. Is there a way to backup/copy/transfer my Audio/Video Profiles so as to not have to reconstruct them for these, or other, issues? 

2. If not, can in please be added as a Feature Request?

I understand that encoder updates can break even these in some cases, but reconstructing previous A/V profiles can be remarkably tedious, error laden, and time consuming resulting in multiple encodes to tune.

Thank you.

**Comment by Dendraspis** [comment]
***TL;DR***
No.

***Long answer***
Meanwhile StaxRip is about 22 years old. It was never designed to be what it is now - meaning that big and complex. 
Back in the days the developers chose the option of saving projects the way we still use with all its advantages and disadvantages. Sadly we only notice the disadvantages, which are mostly issues with the settings, templates and jobs when updating.
Technically it would be possible to avoid some issues by keeping old and abandoned code in StaxRip, but that would make it even more difficult to fix other issues and of course increase the workload and potential to more bugs.
Since `v2.37` I make so called *Update-Releases*, that can be (hopefully) safely used to update the belonging *Main-Release* by simply extract the content into the StaxRip folder you already have:
`v2.42.0` => Main-Release
`v2.42.1` => Update-Release
`v2.42.2` => Update-Release
See also:  [README.md#extraction--usage](https://github.com/staxrip/staxrip?tab=readme-ov-file#extraction--usage)

Here in case of `v2.42` the main issue is that the two audio tracks, that you could find at the bottom of the main window, do not exist anymore. They are completely gone and replaced by a new implementation. So there is no other way but to set everything up from scratch. Additionally the whole handling changed internally, so you have to check and maybe adopt some existing settings anyways.

I have thought many time of changing the way settings are saved, but I don't think there is a better way to do that to keep all advantages.

> I understand that encoder updates can break even these in some cases, but reconstructing previous A/V profiles can be remarkably tedious, error laden, and time consuming resulting in multiple encodes to tune.

Best practice is to run the old and the new version side by side and then go through all settings and transfer everything by hand. So you also notice new and changed settings. When it comes to the encoder settings, you can use the `Import Command Line` feature and double-check the result.


Hope that answers you questions...
